cmd/chartsvc/handler.go

```diff
@@ -200,6 +200,37 @@ func getChartVersionValues(w http.ResponseWriter, req *http.Request, params Para
 	w.Write([]byte(files.Values))
 }
 
+func findLatestChart(name, version, appversion string) []models.ChartLatest {
```

Comment:
What's the reason for the separate function? The `params` parameter in the `resolveRepos` handler does make the handler testable.